assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Tag Manager useExistingTag custom hook.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
Can we change it to 2022?
```suggestion
 * Site Kit by Google, Copyright 2022 Google LLC
```

Reply:
Technically this was re-introduced after having been recently (unexpectedly) deleted, so that can remain using 2021.